### PR TITLE
Assume any locally saved PD data is valid

### PR DESCRIPTION
**Pull request by jewzaam**
i.e. do not verify service id, integration id, or integration key.

### CODE REVIEW

File: pkg/controller/pagerdutyintegration/clusterdeployment_created.go [hunk 1]
```diff
@@ -93,20 +93,54 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 	// To prevent scoping issues in the err check below.
 	var pdIntegrationKey string
 
+	// load configuration
 	err = pdData.ParseClusterConfig(r.client, cd.Namespace, configMapName)
```
Comment (RiRa12621) [inline pkg/controller/pagerdutyintegration/clusterdeployment_created.go:97]:
```suggestion
        err = pdData.ParseClusterConfig(r.client, cd.Namespace, configMapName)
        if err != nil {
          r.reqLogger.Error("Failed to prase cluster config")
        }
```

**Review by mwoodson — APPROVED** [review]
/lgtm

File: pkg/controller/pagerdutyintegration/clusterdeployment_created.go [hunk 2]
```diff
@@ -93,20 +93,56 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
...
```
Comment (RiRa12621) [inline pkg/controller/pagerdutyintegration/clusterdeployment_created.go:97]: Shouldn't we still be doing something with this error?

Comment (jewzaam) [reply pkg/controller/pagerdutyintegration/clusterdeployment_created.go:97]: checking `err` now in the if that follows

**Review by mwoodson — APPROVED** [review]
/lgtm

**Comment by jewzaam** [comment]
https://issues.redhat.com/browse/OSD-5155

**Pull request merged**